lms/static/sass/shared/_header.scss

```diff
@@ -377,6 +388,19 @@ header.global-new {
 
   .nav-global {
     @include float(left);
+    .tab-nav-item{
+      .tab-nav-link{
+        height: 43px;
+        font-size: em(16);
+        color: $gray;
+        &:hover{
```

Comment:
You are talking about the studio home page right?
That tabs styles are not consistent with the one on pattern library tabs. That has the blue underline as the active element, and there are no hover or focus visual cue. Could you double check?